### PR TITLE
Implement EIP-1559

### PR DESCRIPTION
**Pull request by Belteshazzar89**
- Update the Ganache image to use the latest image from the new ganache repository
- Update `send` in the Ethereum driver to take a flag to determine EIP-1559 use, and pass new parameters
- Update default config to use the EIP-1559 flag, with a default value for `maxPriorityFeePerGas`
- Web3 requires `maxFeePerGas`, or it will try to calculate that value itself, though that code crashes. Therefore, we supply that value as well for EIP-1559

### CODE REVIEW

File: packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts [hunk 1]
```diff
@@ -34,6 +35,10 @@ export class EthereumNodeTransactionInfoProvider
       );
       return undefined;
     }
+
+    const maxFeePerGas = formatParameter(transaction.maxFeePerGas);
+    const maxPriorityFeePerGas = formatParameter(transaction.maxPriorityFeePerGas);;
```
Comment (et-taousy) [inline packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts:40]: Je crois tu peux ajouter le type des deux consts:
    
```
    const maxFeePerGas:number = formatParameter(transaction.maxFeePerGas);
    const maxPriorityFeePerGas:number = formatParameter(transaction.maxPriorityFeePerGas);
```

File: packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts [hunk 2]
```diff
@@ -34,6 +35,10 @@ export class EthereumNodeTransactionInfoProvider
       );
       return undefined;
     }
+
+    const maxFeePerGas = formatParameter(transaction.maxFeePerGas);
```
Comment (et-taousy) [inline packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts:39]: Je crois tu peux ajouter le type number.
    
```suggestion
    const maxFeePerGas:number = formatParameter(transaction.maxFeePerGas);
```

**Review by et-taousy — CHANGES_REQUESTED** [review]
Need to just add types baseFeePerGas and maxPriorityFeePerGas move defaultPriorityFeeInGWei

File: packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts (hunk 1 above)
Comment (Belteshazzar89) [reply packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts]: I moved the function calls directly into the values of the constructed object.

File: packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts (hunk 2 above)
Comment (Belteshazzar89) [reply packages/cast/blockchain-driver-eth/src/transaction-info-provider/node/EthereumNodeTransactionInfoProvider.ts]: I moved the function calls directly into the values of the constructed object.

**Review by et-taousy — APPROVED** [review]

**Pull request merged**